### ISSUE TITLE
Animal net duplicates chickens

### ISSUE THREAD

**Issue by CKTheUltimate** [GenDeathrow/Hatchery#11]
When I toss chickens into the nesting pen with the animal net,the chicken is cloned.

**Comment by GenDeathrow** [comment]
Is the the update I pushed last night?

**Comment by GenDeathrow** [comment]
looking into it.. I swear.. I tested everything before I went to bed.. not sure what I changed.,

**Comment by GenDeathrow** [comment]
Fixed.. I know realized why vanilla furnaces have a special boolean to keep inventory. I just had no idea.. I am leaving this open for a while so ppl can see this is fixed.

v0.1.4 ChangeLog *
Issue with Chickens Duping and not being able to place chickens in pen if you break it. 

Any pens that weren't allowing to place a chicken in after this update, just break it and replace it. 

This broke to how vanilla handles blockstate when switching. It considers it as the block is breaking when all I did was change the the state from chicken to no chicken. So it would dump its inventory when adding a chicken.. that would drop a second chicken. 

Sorry for the major bug. I have learned from my mistake.
